fugashi_util.py

```diff
@@ -8,50 +8,61 @@ def mecab_config(com="mecab-config"):
     output = subprocess.check_output([com, "--inc-dir", "--libs-only-L", "--libs-only-l"])
     if not isinstance(output, str):
         output = output.decode("utf-8")
-    return output
+    return output.split('\n'), []
 
-def check_libmecab():
+def mecab_config_windows():
+    ## Windows
     if os.name == 'nt':
         win_mecab_dir = r'C:\mecab'
-        return win_mecab_dir+"\n"+win_mecab_dir+"\nlibmecab\n", [("lib\\site-packages\\", ["{}\\libmecab.dll".format(win_bin_dir)])]
-
-    try:
-        output = mecab_config()
-        return output, []
-    except:
-        pass
+        mecab_details = (win_mecab_dir, win_mecab_dir, 'libmecab')
+        data_files = [("lib\\site-packages\\", ["{}\\libmecab.dll".format(win_bin_dir)])]
+        return mecab_details, data_files
 
+def mecab_config_cygwin():
     base_dir = os.getcwd()
     os.makedirs("build/mecab", exist_ok=True)
     os.chdir("build/mecab")
-
+    ## Cygwin
     if platform.system().startswith("CYGWIN"):
-        rep = "mecab-cygwin64" if platform.machine()=="x86_64" else "mecab-cygwin32"
+        rep = "mecab-cygwin64" if platform.machine() == "x86_64" else "mecab-cygwin32"
         subprocess.run(["git", "clone", "--depth=1", "https://github.com/KoichiYasuoka/"+rep])
         os.chdir(rep)
         subprocess.run(["sh", "-x", "./install.sh", "/usr/local"])
         output = mecab_config("/usr/local/bin/mecab-config")
         return output, []
 
+def mecab_config_debian():
+    ## Debian
+    # XXX how is this different from the debian2 part?
     try:
         subprocess.run(["apt-get", "install", "-y", "libmecab-dev"])
         output = mecab_config("mecab-config")
         return output, []
     except:
         pass
 
+def mecab_config_debian2():
```

Comment:
Please clarify what platform this is for.

Reply:
mecab_config_debian() is for root install, and mecab_config_debian2() is for user install. However, in your re-writing of `mecab_config`, `replace` for strings won't work now...